src/backend/app/projects/project_routes.py

```diff
@@ -240,20 +243,25 @@ async def delete_project(
 @router.post("/create_project", response_model=project_schemas.ProjectOut)
 async def create_project(
     project_info: project_schemas.ProjectUpload,
-    current_user: AuthUser = Depends(login_required),
+    org_user_dict: db_models.DbUser = Depends(org_admin),
```

Comment:
Both org_id and project_id are specified as route params here, so I will just update the frontend so the org id is passed in the route, not the json.
`API/create_project?org_id=1`

Reply:
![image](https://github.com/hotosm/fmtm/assets/78538841/d1eb98a3-7cf3-4df5-b617-6431e6f06243)

Generally the org_id should be passed. But it's also possible to pass the project_id of an existing project in the org, and the org_id will be extracted too.

Reply:
In create_project we won't have project_id right?, so yeah org_id can be passed from the frontend.

Reply:
Yep org_id will be passed.

But it is possible to specify **another** project id from a project in the same organisation, and the code would still identify which org you want to create the project in (not that useful, but good to know).

Reply:
is it good idea to use project_id of already existing project just for the organisation? since it wont work if there are not any existing projects in the organisation. i think its better to follow consistent idea to pass org_id